components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/OIDCSessionStateManager.java

```diff
@@ -42,6 +42,19 @@
      * @param response
      * @return Cookie
      */
+    @Deprecated
```

Comment:
let's add description why are we deprecating and what is the new method to be used.

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1622/commits/7d8e344901ed2569cd28733bed3f2a770a982271